### ISSUE TITLE
does not start in the window

### ISSUE THREAD

**Issue by sergeshat** [kolton/d2bot-with-kolbot#2046]
Created a character, but it runs only in full screen mode and can not configure.All others run in the window

**Comment by DS1337** [comment]
In your D2Bot Profile u must paste "-w" in "parameters!
Then your game starting in window-mode

**Comment by sergeshat** [comment]
I know that.-w is registered, I tried everything(changed the name on the working profiles to it), but it works in full screen mode.

**Comment by DS1337** [comment]
Did u try to copy/clone the working profiles and just chance login stuff etc.?

never hear about some problem like this...i mean, the others works but 1 not?
hmmm..

**Comment by sergeshat** [comment]
Yes, I tried working profiles, changed the name, login, password of this character.It started, worked fine, but only in full screen mode.To test on the same account created another character, changed the name to it-earned in the window

**Comment by NipperDipper** [comment]
Did you ever find a fix for this issue? I'm having the same problem currently.
